### PR TITLE
Merge pull request #1008 from Avaiga/doc/#404-close-databricks

### PR DESCRIPTION
**Pull request by FlorianJacta**
Make Databricks integration an Enterprise feature.

### CODE REVIEW

File: docs/tutorials/integration/4_databricks/index.md [hunk 1]
```diff
@@ -8,226 +8,17 @@ img: 4_databricks/images/databricks.png
 
 Integrating Taipy with Databricks is a great way to build user-centric applications. 
 Taipy GUIs and SDM (Scenario and Data Management) will transform your Databricks models
```
Comment (jrobinAV) [inline docs/tutorials/integration/4_databricks/index.md]:
```suggestion
Taipy will transform your Databricks models 
```

File: docs/tutorials/integration/4_databricks/index.md [hunk 2]
```diff
@@ -8,226 +8,17 @@ img: 4_databricks/images/databricks.png
 
 Integrating Taipy with Databricks is a great way to build user-centric applications.
```
Comment (jrobinAV) [inline docs/tutorials/integration/4_databricks/index.md]: THis first sentence is wrong to me and does not reflect the motivation.

What about: 
Integrating a Taipy application with Databricks allows you to quickly build a web application on top of your Databricks notebooks. You can easily put your algorithms into end-users hands.

**Comment by FlorianJacta** [comment]
The PR has already been approved for develop. Should I go back to the develop and close this PR? @jrobinAV

**Comment by jrobinAV** [comment]
I would make the change on both develop and release/3.1.

By the way. What about release/3.0?

**Comment by FlorianJacta** [comment]
Is the correct way not to cherry pick each time?

**Comment by jrobinAV** [comment]
yep. Cherry pick is the way, but you can do it in both ways. It is better to do it from develop to release/x.y, but you can do it the other way around.

**Review by jrobinAV — APPROVED** [review]

**Pull request merged**